test/topology_custom/test_mv_loss_attrs.py

```diff
@@ -0,0 +1,97 @@
+#
+# Copyright (C) 2024-present ScyllaDB
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+#
+from test.pylib.manager_client import ManagerClient
+
+import pytest
+import time
+import logging
+import glob
+import os
+
+from test.topology.util import reconnect_driver
+from test.pylib.util import wait_for_view
+
+logger = logging.getLogger(__name__)
+
+@pytest.mark.asyncio
+async def test_mv_loss_attrs_exception(manager: ManagerClient) -> None:
+    node_count = 2
+    servers = await manager.servers_add(node_count)
+    cql, _ = await manager.get_ready_cql(servers)
+    await cql.run_async("CREATE KEYSPACE ks WITH replication = {'class': 'NetworkTopologyStrategy', 'replication_factor': 2} AND tablets = {'enabled': false};")
+    # Create a table with non-primary key columns as collections
+    await cql.run_async("""CREATE TABLE ks.tab (base_key int, view_key int, ":attrs" map<text, text>, PRIMARY KEY (base_key, view_key)) WITH gc_grace_seconds = 1 AND tombstone_gc = {'mode': 'timeout'};""")
+    # Create a MV
+    await cql.run_async("CREATE MATERIALIZED VIEW ks.mv_cf_view AS SELECT * FROM ks.tab WHERE view_key IS NOT NULL and base_key IS NOT NULL PRIMARY KEY (view_key, base_key);")
+
+    await wait_for_view(cql, 'mv_cf_view', node_count)
+
+    # Simulate Alternator to perform the operation of deleting first and then writing
+    await cql.run_async("DELETE FROM ks.tab WHERE base_key = 5 AND view_key = 5;")
+    await cql.run_async('INSERT INTO ks.tab (base_key, view_key, ":attrs") VALUES (5, 5, {\'attr1\': \'aaaaa\'});')
+
+    # waite for gc_grace_seconds
+    time.sleep(10)
```

Comment:
gc_grace_seconds = 1, why sleep for 10 seconds here?

Also, does the garbage-collection method `tombstone` connected with how the bug reproduces?

Reply:
Used for self-testing, has been modified to `time.sleep(1)`.

It doesn't matter, this bug will occur regardless of `timeout` or `gc_mode=repair`.